src/main/java/uk/gov/companieshouse/limitedpartnershipsapi/service/LimitedPartnerValidator.java

```diff
@@ -113,17 +114,31 @@ private void validatePrivateFundPartnershipContributions(String contributionCurr
     }
 
     private void validateStandardPartnershipContributions(String contributionCurrencyValue, Currency contributionCurrencyType, boolean hasContributionSubTypes, BindingResult bindingResult) {
-        if (contributionCurrencyValue == null || contributionCurrencyValue.isBlank() || contributionCurrencyValue.equals("0.00")) {
+        if (contributionCurrencyValue == null || contributionCurrencyValue.isBlank()) {
```

Comment:
Based on my comment on the other PR I was only expecting this line to change. Something like:

`if (contributionCurrencyValue == null || (BigDecimal.ZERO.compareTo(new BigDecimal(contributionCurrencyValue)) == 0)) {`

Reply:
I know but we remove the regex validation then we still need to validate to exclude the other formats specified on the ticket.

Reply:
Yep, you've done that now by re-including the regex in the DTO (which will validate the formats).

Reply:
We need to try catch and handle it or we'll validate values with invalid character twice and get a 500 error the second time.

Reply:
Sorry, I now see that an invalid format causes creation of the BigDecimal value to fail, so see what you've done here.